mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -905,24 +923,6 @@ The following arguments are supported:
   (Required)
   `cidr_block` must be specified in CIDR notation.
 
-* `data_retention_config` -
```

Comment:
why is this block moved?

Reply:
For Composer 2, It was mistakenly intended under cidr_blocks while it is part of config: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb#L736